src/react-apps/applications/shared/src/theme/altinnAppTheme.tsx

```diff
@@ -134,4 +134,4 @@ const AltinnAppTheme = {
   },
 };
 
-export default AltinnAppTheme;
+export default altinnAppTheme;
```

Comment:
any reason for this change? Or is it just for uniformity?

Reply:
See we use capitalized letters on the others, changing back :)